gatsby-config.js

```diff
@@ -193,34 +188,47 @@ module.exports = {
             options: {
               output: '/sitemap-all.xml',
               serialize: ({ site, allSitePage }) => {
-                const allPages = allSitePage.edges.map((edge) => edge.node);
+                const latestPages = allSitePage.edges.map((edge) => ({
+                  url: `${site.siteMetadata.siteUrl}${edge.node.path}/`,
+                  changefreq: 'daily',
+                  priority: 0.7,
+                }));
 
-                return allPages.reduce((acc, page) => {
-                  if (page.path.startsWith('/docs')) {
-                    [...versions.stable, ...versions.preRelease].forEach(({ label, string }) => {
-                      let pagePath = page.path;
-                      if (label !== 'latest') {
-                        const parts = page.path.split('/');
-                        parts.splice(2, 0, string);
-                        pagePath = parts.join('/');
-                      }
+                const nonLatestDocsPages = [];
+                versionsWithToc.forEach(({ string, toc }) => {
+                  [...siteMetadata.coreFrameworks, ...siteMetadata.communityFrameworks].forEach(
+                    (framework) => {
+                      const createDocsPages = (tocItems, pathPrefix) => {
```

Comment:
There's another `createDocsPages` function in `gatsby-node.js` to actually generate the page. Using the same name here threw me off. Could we rename this to `createDocsPagesInfo` or something.